.github/workflows/release.yaml

```diff
@@ -1,11 +1,9 @@
 name: Helm Charts Release
 
-on: 
+on:
   push:
     branches:
-      - "!*"
-    tags:
-      - "v*"
+      - main
```

Comment:
Should we not release on tag?

Reply:
was hoping for a <chart>-<version> pattern

Reply:
As far as I know, `chart-releaser-action` creates the tag automatically for us in the release using the chart name and new version to create the - pattern. I'm waiting for the first run and release to see it working though.

Reply:
Let me know if you prefer us to create the tags and release on these tags.  I recommend the automatic way as it looks like a standard practice among several helm chart repos.

Reply:
All good, didn't know it works this way. Thanks!